IntercomDotnet/Client.cs

```diff
@@ -15,9 +15,9 @@ public class Client
         public string UserName { get; set; }
         public string Password { get; set; }
 
-        public double GetUnixTimestamp(DateTime inputTime)
+        public long GetUnixTimestamp(DateTime inputTime)
         {
-            return (inputTime - new DateTime(1970, 1, 1).ToUniversalTime()).TotalSeconds;
+            return (long)inputTime.ToUniversalTime().Subtract(new DateTime(1970, 1, 1, 0, 0, 0)).TotalSeconds;
```

Comment:
I think you either need to have `ToUniversalTime()` on both `inputTime` and `new DateTime(1970, 1, 1)`, or on neither. Isn't the DateTime going to get initialised in the running system's timezone? And input time will now be changed to UTC time?

Also I just checked how I haven't hit this issue (the truncating issue), and I have a similar helper method in my libs which does:

```csharp
public int GetUnixTimestamp(DateTime dateTime)
{
    return (int)Math.Round((dateTime - new DateTime(1970, 1, 1).ToUniversalTime()).TotalSeconds);
}
```

So I think I had hit this problem before, but I fixed it on my caller, but not the client implementation.